### DISCUSSION TITLE
An interesting article on .NET MAUI worth looking at

### DISCUSSION THREAD

**Discussion by dg2k** [dotnet/maui#19552] [General]
The article is at LinkedIn [here](https://www.linkedin.com/pulse/taming-microsoft-net-maui-beast-multiplatform-app-demessie-girma-qrmwe).

**Comment by mikkpaasuke** [comment]
Excellent, thanks for sharing.

**Comment by ziomek64** [comment]
> The article is at LinkedIn [here](https://www.linkedin.com/pulse/taming-microsoft-net-maui-beast-multiplatform-app-demessie-girma-qrmwe).

Did you write it?
